src/components/SectionInfo.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import styled from "styled-components";
+import Button from "./Button";
+import InformationImage from "../images/section-2.png";
+
+function SectionInfo() {
+  return (
+    <Wrapper>
+      <InfoImage>
+        <Image src={InformationImage} />
+      </InfoImage>
+      <InfoWrapper>
+        <Info>
+          <InfoTitle>A new way to explore the</InfoTitle>
+          <InfoTitle>world</InfoTitle>
+          <Information>
+            <p>
+              For decades travellers have reached for Lonely Planet books when
+              looking to plan and execute their perfect trip, but now, they can
+              also let Lonely Planet Experiences lead the way
+            </p>
```

Comment:
Burada Information u styled.p olarak değiştirmen yeterli, icine tekrar p tagi eklemene gerek yok

`
<Information>For decades travellers have reached for Lonely Planet books when
               looking to plan and execute their perfect trip, but now, they can
               also let Lonely Planet Experiences lead the way
</Information>
`